client/src/components/Form/components/formSections/BekreftelseSection.tsx

```diff
@@ -34,48 +37,43 @@ type BekreftelseSectionProps = {
 };
 
 const BekreftelseSection = ({ section, setSchema, schema, errors, validate }: BekreftelseSectionProps) => {
+    const [sykmelder, setSykmelder] = useState<Sykmelder | undefined | null>(undefined);
+    const [isLoading, setIsloading] = useState<boolean>(false);
+    const [error, setError] = useState<Error | null>(null);
+
+    // GET information about sykmelder on every schema.hpr change
+    useEffect(() => {
+        // Number must be in synch with validationFuncitons.hpr in validation.ts
+        if (schema.hpr?.length === 7) {
+            setIsloading(true);
+            setError(null);
+            fetch(`/backend/api/v1/sykmelder/${schema.hpr}`, { credentials: 'include' })
+                .then((res) => {
+                    if (res.ok) {
+                        return res.json();
+                    } else {
+                        throw new Error('Fant ikke behandler med hprNummer: ' + schema.hpr);
+                    }
+                })
+                .then((jsonResponse) => {
+                    return iotsPromise.decode(Sykmelder, jsonResponse);
+                })
+                .then((sykmelder) => {
+                    setSykmelder(sykmelder);
+                })
+                .catch((error) => {
+                    console.log(error);
+                    setSykmelder(null);
+                    setError(error);
+                })
+                .finally(() => setIsloading(false));
+        } else {
+            setSykmelder(null);
+        }
+    }, [schema.hpr]);
+
```

Comment:
Det er mulig at en bedre brukeropplevelse her ville vært å ha en boks ved siden av HPR-inputfeltet som bruker må trykke på for å laste inn info om legen.

Nå er det slik at med en gang bruker har skrevet inn 7 siffer så hentes data og fokusmarkøren flyttes. Dersom bruker har skrevet feil siffer så blir det smått tungvint å markere inputfeltet på nytt og rette på tallet.

Igjen, småpirk som gjerne kan droppes i denne omgang.

Reply:
jeg liker forslaget. Må høre med bendik her

Reply:
Har hatt en diskusjon på dette, og vi beholder det slik.